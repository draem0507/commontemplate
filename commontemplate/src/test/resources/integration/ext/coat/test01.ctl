非标准起始文字
some text
<!-- 注释块 -->
<html>
	<input type="text" ct:if="yy">
	<something>
	asdfasdf;<"&<,>&lt;&gt;
	<!-- 注释块 -->
    <body>
        <table ct:if="users != null && users.size> 0" ct:for="xx" border="1">
            <tr ct:for="user : users">
                <td><span ct:if="for.index > 0" ct:out="for.index + 1">1</span></td>
                <td><span ct:out="user.name">james</span></td>
                <td><span ct:out="user.coins">2.00</span></td>
            </tr>
        </table>
    </body>
</html>
<tag ct:if="xx">
	<tag ct:if="yy"/>
</tag>
some text
非标准结束
