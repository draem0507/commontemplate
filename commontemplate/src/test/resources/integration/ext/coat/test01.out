非标准起始文字
some text
<html>
	$if{yy}<input type="text" ct:if="yy">$end
	<something>
	asdfasdf;<"&<,>&lt;&gt;

    <body>
        $if{users != null && users.size > 0}<table ct:if="users != null && users.size > 0" border="1">
            $for{user : users}<tr ct:for="user : users">
                <td>$out{for.index + 1}<span ct:out="for.index + 1">1</span>$end</td>
                <td>$out{user.name}<span ct:out="user.name">james</span>$end</td>
                <td>$out{user.coins}<span ct:out="user.coins">2.00</span>$end</td>
            </tr>$end
        </table>$end
    </body>
</html>
$if{xx}<tag ct:if="xx">
	$if{yy}<tag ct:if="yy"/>$end
</tag>$end
some text
非标准结束
